kolibri/plugins/coach/assets/src/views/index.vue

```diff
@@ -8,7 +8,7 @@
         <top-nav/>
       </template>
 
-      <div v-if="isCoach || isAdmin || isSuperuser">
+      <div v-if="userCanAccessPage">
         <component :is="currentPage"/>
       </div>
       <auth-message v-else authorizedRole="adminOrCoach" />
```

Comment:
@indirectlylit note that the auth-message component will need to be cleaned up wrt `superuser`, but as you want to do that in one sweep, I think we'll leave this as is for now.